app/ts/background/background.ts

```diff
@@ -475,17 +477,17 @@ export function sendMessageToContentScript(websiteTabConnections: WebsiteTabConn
 	})
 }
 
-export async function handleContentScriptMessage(websiteTabConnections: WebsiteTabConnections, socket: WebsiteSocket, request: InterceptedRequest, website: Website) {
+export async function handleContentScriptMessage(websiteTabConnections: WebsiteTabConnections, socket: WebsiteSocket, request: InterceptedRequest, website: Website, activeAddress: bigint | undefined) {
```

Comment:
Would optional more accurately represent the intent of this parameter, or is an explicit `undefined` expected?
```suggestion
export async function handleContentScriptMessage(websiteTabConnections: WebsiteTabConnections, socket: WebsiteSocket, request: InterceptedRequest, website: Website, activeAddress?: bigint) {
```

Reply:
I guess with better wording. I want the active address be mandatory when it exists, and it would be a bug to leave it out when it exists